packages/components/src/modal/style.scss

```diff
@@ -77,7 +82,7 @@
 	left: 0;
 
 	.components-modal__header-heading {
-		font-size: 1rem;
+		font-size: 1.2rem;
```

Comment:
It would be nice for things like this to be systematised, I'm aware of the experimental `Text` component, but it's not widely used. Perhaps there's a better way?

Reply:
I'd personally love to see the [`Text`](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-text--default) and [`Heading`](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-heading--default) components user more often.

I'm not sure if we need to first assess the type scale (and the rest of the styles), but otherwise I think we should slowly try to adopt them in other components, find out if they match our needs and make any necessary changes, before then rolling them out more widely across Gutenberg